x/ethbridge/keeper/msg_server.go

```diff
@@ -131,18 +127,25 @@ func (srv msgServer) Burn(goCtx context.Context, msg *types.MsgBurn) (*types.Msg
 		return nil, fmt.Errorf("token metadata not available for %s", msg.DenomHash)
 	}
 
-	prophecyID, err := srv.Keeper.ProcessBurn(ctx, cosmosSender, account.GetSequence(), msg, tokenMetadata)
+	doublePeg := tokenMetadata.NetworkDescriptor != msg.NetworkDescriptor
```

Comment:
Isn't this always true for burns?

Reply:
If the token burned and go back to its orignal network, it is not double peg.

Reply:
right, don't know what I was thinking.